ext/snmp/tests/snmp_set_string_output_format.phpt

```diff
@@ -0,0 +1,32 @@
+--TEST--
+Function snmp_set_string_output_format
+--CREDITS--
+Larry Adams
+--SKIPIF--
+<?php
+require_once(__DIR__.'/skipif.inc');
+if (!function_exists('snmp_set_string_output_format')) die('skip This function is only available if using NET_SNMP');
```

Comment:
As far as I can tell, a separate check for the function is not needed here, just the `skipif.inc` include is enough. If the extension is loaded, then the function will always exist.

Reply:
As I said before, I copied whole cloth from a similar function.  Give it some thought as making that decision here should/will impact others as well.